addons/mk6mortar/stringtable.xml

```diff
@@ -512,5 +545,20 @@
             <Chinesesimp>[ACE] 82mm预设弹药箱</Chinesesimp>
             <Chinese>[ACE] 82mm預設彈藥箱</Chinese>
         </Key>
+        <Key ID="STR_ACE_Mk6Mortar_propelling_charge_name">
+            <English>Propelling Charge</English>
+        </Key>
+        <Key ID="STR_ACE_Mk6Mortar_propelling_charge_description">
+            <English>For use with 82mm mortar ammunition</English>
+        </Key>
+        <Key ID="STR_ACE_Mk6Mortar_useChargeSystem_DisplayName">
+            <English>Use Charge System</English>
+        </Key>
+        <Key ID="STR_ACE_Mk6Mortar_useChargeSystem_Description">
+            <English>Requires 82mm mortar shells to have their charges altered to achieve the desired range. Will affect AI</English>
+        </Key>
+        <Key ID="STR_ACE_Mk6Mortar_ammunition_action">
+            <English>Prepare Ammunition</English>
```

Comment:
Different from https://github.com/acemod/ACE3/pull/6577/files#diff-828b9969fd2828d749a5b198f91748baR278 ?

Reply:
Im not following; preparing ammunition and to prepare ammunition is different

Reply:
Yes that was the question. Fine then.